src/components/ConversationSettings/ConversationSettingsDialog.vue

```diff
@@ -37,6 +37,7 @@
 			<NcAppSettingsSection id="conversation-settings"
 				:name="selfIsOwnerOrModerator ? t('spreed', 'Moderation') : t('spreed', 'Setup overview')">
 				<ListableSettings v-if="!isNoteToSelf && !isGuest" :token="token" :can-moderate="canFullModerate" />
+				<MentionsSettings v-if="!isNoteToSelf && !isGuest" :token="token" :can-moderate="selfIsOwnerOrModerator" />
```

Comment:
`:can-moderate="canFullModerate"`

Using `selfIsOwnerOrModerator` will also show this setting in one-to-one conversations.

Reply:
fixed now, I also removed `isGuest` check, review needed :)